### PR TITLE
PLSVC-1545 Initial commit of skeleton repo

### PR DESCRIPTION
**Pull request by biwwywiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# navient-sso-token-service
\ No newline at end of file
+# navient-sso-token-service
+
+[![Build Status](https://jenkins-external.tools.earnest.com/buildStatus/icon?job=meetearnest/navient-sso-token-service/master)](https://jenkins.tools.earnest.com/blue/organizations/jenkins/meetearnest%2Fnavient-sso-token-service/activity/)
+
+Service for boarding Earnest loans to external loan servicing systems, such as Navient.
```
Comment (khomni) [inline README.md]: Hold-over from boarding-service

File: src/services/swagger.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  register: require('hapi-swagger'),
+  options: {
+    documentationPath: '/v1/documentation',
+    swaggerUIPath: '/v1/swaggerui/',
+    jsonPath: '/v1/swagger.json',
+    info: {
+      title: 'Boarding Service API'
```
Comment (khomni) [inline src/services/swagger.js]: boarding-service holdover

File: vault-data/boarding-service.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (khomni) [inline vault-data/boarding-service.json]: filename is a boarding-service holdover

Comment (biwwywiu) [reply vault-data/boarding-service.json]: @khomni Thanks for spotting. Updated.

File: src/services/swagger.js (hunk 2 above)
Comment (biwwywiu) [reply src/services/swagger.js]: @khomni Thanks for spotting. Updated.

File: README.md (hunk 1 above)
Comment (biwwywiu) [reply README.md]: @khomni Thanks for spotting. Updated.

**Pull request merged**